ossfuzz/standaloneengine.cc

```diff
@@ -0,0 +1,74 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "testinput.h"
+
+/**
+ * Main procedure for standalone fuzzing engine.
+ *
+ * Reads filenames from the argument array. For each filename, read the file
+ * into memory and then call the fuzzing interface with the data.
+ */
+int main(int argc, char **argv)
+{
+  int ii;
+  for(ii = 1; ii < argc; ii++)
```

Comment:
Please use a space between statement keywords and `(`.  Ie. `for(` => `for (`

Note.  This only applies to `if` `for`, etc.  Functions and function calls should _not_ have a space before the `(`.